spec/models/service_template_ansible_playbook_spec.rb

```diff
@@ -237,7 +237,7 @@
     end
   end
 
-  describe '#destroy' do
+  describe '#delete_job_templates' do
```

Comment:
I think we can make `delete_job_templates` private, and the spec still tests on `#destroy`. This way we make sure job templates are deleted when destroying a service template.